scripts/rollup/header.js

```diff
@@ -15,14 +15,22 @@ function getProvidesHeader(hasteFinalName) {
 `;
 }
 
-function getUMDHeader(filename, reactVersion) {
+function getHeader(filename, reactVersion) {
   return `/**
- * ${filename} v${reactVersion}
+ * React v${reactVersion}
```

Comment:
We should still include the filename IMO. Maybe make it the first line, and then the rest?